### ISSUE TITLE
Editing an outbox message drop inReplyToMessageId

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/mail#6499]
### Steps to reproduce

1. Open an existing message
2. Click *Reply*
3. Click *Send*
4. Go to the outbox
5. Open the message again
6. Click *Send*
7. (Possibly repeat step 5+6)
8. Wait for the message to send

### Expected behavior

1) The reply message has an `in-reply-to` header that points to the original message
2) The original message is marked as answered

### Actual behavior

Neither of the two.

### Mail app version

v1.12.0

### Additional info

Similar flow as https://github.com/nextcloud/mail/issues/6494 but this time it's not the server that drops the attribute. It looks like it gets lost on the front-end.

**Comment by ChristophWurst** [comment]
@GretaD could you please trace where the attribute gets lost so we can narrow down the cause?

**Comment by ChristophWurst** [comment]
@GretaD any findings?

**Comment by GretaD** [comment]
No, nothing useful, while using the vue tools, i dont see any inReplyToMessageId anywhere. Even when i try it on a correct msg that has the in-reply-to on source. Maybe smth on backend? I dont know where else to look as this doesn't seem to be a regression

**Comment by ChristophWurst** [comment]
![Bildschirmfoto vom 2022-06-01 09-18-32](https://user-images.githubusercontent.com/1374172/171349331-3d6e5435-1f75-40f1-9828-f2ab3a241ac2.png)

`inReplyToMessageId` is there for me. Could you check again?
